README.md

```diff
@@ -18,7 +18,7 @@ Repository to describe, develop, document and test the Identity And Consent Mana
 ## Meetings
 * Meetings are held virtually
 * Schedule: bi-weekly, Wednesday, 4 PM CET/CEST (15:00 UTC, 14:00 UTC during European DST). For next call, see minutes of [previous meetings](documentation/MeetingMinutes) 
-* Meeting link: [Microsoft Teams Meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_NWUxYjE5YjQtMTU2OS00Y2IyLWFhNTYtM2E4OGZkZjYxY2Y0%40thread.v2/0?context=%7b%22Tid%22%3a%229744600e-3e04-492e-baa1-25ec245c6f10%22%2c%22Oid%22%3a%22ec8dd69b-01fe-4d41-a294-c2927b548e27%22%7d)
+* For meeting details please use the [Camara calendar](https://lists.camaraproject.org/calendar) or import this [ICS](https://lists.camaraproject.org/g/sp-icm/ics/invite.ics?repeatid=57069) file into your favorite calendar app.
```

Comment:
I'm not sure that the link to the invite to the .isc file is stable (might be). In TSC and other sub projects we used the same link which is on the [wiki page](https://wiki.camaraproject.org/pages/viewpage.action?pageId=393475) for "Register/Join". It allows to ask for an invitation. 

```suggestion
* Meetings are held virtually: [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3) 
```